cloudflare/data_source_zone_dnssec.go

```diff
@@ -0,0 +1,98 @@
+package cloudflare
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceCloudflareZoneDNSSEC() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceCloudflareZoneDNSSECRead,
+
+		Schema: map[string]*schema.Schema{
+			"zoneid": {
```

Comment:
We should keep the consistency of `zone_id` here to mimic what we do in other places.

```suggestion
			"zone_id": {
```

Reply:
Fixed